src/components/pages/ForgotPassword.js

```diff
@@ -0,0 +1,65 @@
+/* eslint-disable react/destructuring-assignment */
+import React from 'react';
+import { connect } from 'react-redux';
+import PropTypes from 'prop-types';
+import { compose } from 'redux';
+import { withRouter } from 'react-router-dom';
+import { ResetAction } from '../../actions/resetPasswordAction';
+import Input from '../global/Input';
+import Button from '../global/Button';
+
+export class Forgotpassword extends React.Component {
+    state = {
+      email: '',
+      loading: false,
+    };
+
+  handleChange = (e) => {
+    const { name, value } = e.target;
+    this.setState({
+      [name]: value,
+    });
+  };
+
+handleSubmit = (e) => {
+  e.preventDefault();
+  const { props, state } = this;
+  this.setState((prevState) => ({
+    loading: !prevState.loading,
+  }));
+
+  const timeS = performance.now();
```

Comment:
What is this function doing?

Reply:
it is fro loading.it helps to check the time to get the data .